src/jira/client/jira-client.ts

```diff
@@ -356,6 +357,14 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		appProperties: {
+			create: (isConfiguredState: string) =>
+				instance.put(`/rest/atlassian-connect/1/addons/${getAppKey()}/properties/isConfigured`, {
```

Comment:
Can you please update the camelcased `isConfigured` to `is-configured`? The property is now hard-coded and I think hard-coded strings are hyphenated. 
`is-configured` is being used in both JBE and JFE. 
And also, can we update the version `1` to `latest`?